remote-workstations/new-agent-vm/Install-PCoIPAgent.ps1

```diff
@@ -351,8 +382,13 @@ Configuration InstallPCoIPAgent
             GetScript  = { return 'registration'}
             
             TestScript = { 
-                cd "C:\Program Files (x86)\Teradici\PCoIP Agent"
-                 $ret = & .\pcoip-validate-license.ps1
+                $path = "C:\Program Files (x86)\Teradici\PCoIP Agent"
```

Comment:
Why not use the RegKey path like in the Test-Scipt instead of doing a mix of RegPath and InstallPath checks to pick the install path